doc/sections/ex-gwf-curvilinear-90.tex

```diff
@@ -0,0 +1,44 @@
+
+The radial band numbering starts at 1 for the inner most band and the column number starts at 1 for the column closes to the x-axis. For MODFLOW 6, the DISV package identifies cells by the CELL2D number ($icell2d$). For this example, $icell2d$ starts at radial band 1, column 1 and increases sequentially in the column direction, then the radial direction (fig.~\ref{fig:ex-gwf-curve-90-grid}).
+
+All vertices between two radial bands have the same radial distance from the axis origin. The inner- and outer most vertices have a radial distance of 4 $ft$ and 20 $ft$, respectively. The radial distance between band vertices is 1 $ft$ for a total of 16 curvilinear radial bands. The radial bands are discretized in 5$^{\circ}$ increments for a total of 18 columns per band. The single model layer is 10 $ft$ thick with a transmissivity of 0.19 $ft^2/day$. A constant head boundary condition is assigned to the inner and outer most radial bands at 10 $ft$ and 3.334 $ft$, respectively. The remaining model properties are summarized in table~\ref{tab:ex-gwf-curvilinear-90-01}.
+
+\begin{StandardFigure}{
+                                     Model grid used for the 90$^{\circ}$ curvilinear vertex model.
+                                     Constant-head cells are marked in blue.
+                                     The inner constant head is 10 $ft$ and the outer constant head is 3.33 $ft$.
+                                     Cell numbers are shown inside each model cell.
+                                     Vertices are also numbered and are shown in red.
+                                     }{fig:ex-gwf-curve-90-grid}{../figures/ex-gwf-curve-90-grid.png}
+\end{StandardFigure}
+
+
+% add static parameter table(s)
+\input{../tables/ex-gwf-curvilinear-90-01.tex}
+
+% for examples without scenarios
+\subsection{Example Results}
+The curvilinear vertex model is solved using one, steady state, stress period and compares it to the analytical solution (eqn. 5.4 in \cite{crank1975diffusion}). Figure~\ref{fig:ex-gwf-curve-90-head} presents the MODFLOW 6 simulated head and flow lines for all model cells. Figure~\ref{fig:ex-gwf-curve-90-obs-head} presents the head solution along column 9 from all radial bands ($icell2d\colon 9, 27, 45, \ldots, 261, 279$) and compares it to the analytical solution. The MODFLOW 6 head solution occurs at the cell center rather than the edge, so the radial distance of the inner most head solution is 4.5 $ft$ instead of 4 $ft$. The MODFLOW 6 results are nearly identical to the analytical solution with a relative error of 0.05\% and root mean square error of 0.0037 $ft$.
```

Comment:
```suggestion
The curvilinear vertex model is solved using one, steady state, stress period and is compared to the analytical solution (eqn. 5.4 in \cite{crank1975diffusion}). Figure~\ref{fig:ex-gwf-curve-90-head} presents the MODFLOW 6 simulated head and flow lines for all model cells. Figure~\ref{fig:ex-gwf-curve-90-obs-head} presents the head solution along column 9 from all radial bands ($icell2d\colon 9, 27, 45, \ldots, 261, 279$) and compares it to the analytical solution. The MODFLOW 6 head solution occurs at the cell center rather than the edge, so the radial distance of the inner most head solution is 4.5 $ft$ instead of 4 $ft$. The MODFLOW 6 results are in agreement with the analytical solution with a relative error of 0.05\% and root mean square error of 0.0037 $ft$.
```